Bot name - @HelpixFindBot. Висит на PythonAnywhere, пишет в базу данных пользователей, текст их сообщений и дату получения сообщения.
P.S. 01.04 - я не знаю, это первоапрельская шутка от PythonAnywhere или что, но у меня полетел весь код. Он работал. Он абсолютно точно работал. Но попытки перезапустить бота проваливаются стабильно. Хотя админка стоит. Интересно. Буду отписывать сюда по мере фиксов.
02.04. - я выяснил, что произошло. При запуске "по умолчанию" (то есть python manage.py bot) он умудрился использовать версию питона 2.7, что приводило к веселого рода ошибкам. Далее у меня почему-то слетела библиотека pytelegrambotapi, которую я использовал для бота (такое ощущение, что они экстренно потерли оттуда python3.9, ибо вроде как до этого все бегало на нем, и за ним же полетела и библиотека) и некоторое время не хотела устанавливаться (errno 13 permission denied), благо это пофиксить получилось. Так что сейчас вроде как работает, но что это за дичь была в исполнении PythonAnywhere, мне понять так и не удалось.
